js/main.js

```diff
@@ -94,6 +94,20 @@
     } else {
       $scope.year = date.getFullYear();
     }
+    $scope.shiftMonth = function(val, shift) {
+      var j, len, object, ref, date;
+      ref = $scope.date_options_month;
+      for (j = 0, len = ref.length; j < len; j++) {
+        object = ref[j];
+        if ((object.value === val.toLowerCase()) || (object.name.toLowerCase() === val.toLowerCase())) {
+          var date = new Date($scope.year, j, 01); // First day of prev or next month (move by 'shift' months)
+          date.setMonth(date.getMonth() + shift);
+          $scope.year = date.getFullYear();
+          $scope.month = $scope.date_options_month[date.getMonth()].value;  
+          $scope.monthChange();
+        }
+      }
+    };    
```

Comment:
One wouldn't think so, but `main.js` is actually a generated file, based on `js/cs/main.coffee`. So to make this work and not be overwritten next time someone (like me) runs `grunt`, it needs to be moved there and converted to CoffeeScript :coffee: